### PR TITLE
Cpix

### PR DESCRIPTION
**Pull request by rogerpales**

### CODE REVIEW

File: docs/source/VUDRM-CPIX.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+> Available DRM systems [`fairplay`, `playready`, `widevine`].
+
+## Endpoints
+
+### Default
+
+Get a CPIX 2.1 document with all DRM systems `<client-name>` is entitled to use.
+
+#### optional query params:
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
#### Optional query params:
```

File: docs/source/VUDRM-CPIX.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+* `drm` : comma separated list of DRM systems to be included included on the response.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
* `drm` : comma separated list of DRM systems to be included included in the response.
```

File: docs/source/VUDRM-CPIX.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+    </cpix:DRMSystem>
+  </cpix:DRMSystemList>
+</cpix:CPIX>
+```
+
+### Decrypt
+
+Get a CPIX 2.1 document with only a single content key.
+
+#### optional query params:
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
#### Optional query params:
```

File: docs/source/VUDRM-CPIX.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+
+* `drm` : comma separated list of DRM systems to be included included on the response.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
* `drm` : comma separated list of DRM systems to be included included in the response.
```

File: docs/source/VUDRM-CPIX.md [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+    </cpix:ContentKey>
+  </cpix:ContentKeyList>
+</cpix:CPIX>
+```
+
+### Multikey
+
+Get a CPIX 2.1 document that uses separate keys for video and audio tracks. Only supported for DASH with PlayReady and Widevine.
+
+#### optional query params:
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
#### Optional query params:
```

File: docs/source/VUDRM-CPIX.md [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+
+* `drm` : comma separated list of DRM systems to be included included on the response.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
* `drm` : comma separated list of DRM systems to be included included in the response.
```

File: docs/source/VUDRM-CPIX.md [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+    <cpix:ContentKeyUsageRule kid="22222222-2222-2222-22222-22222222222">
+      <cpix:AudioFilter></cpix:AudioFilter>
+    </cpix:ContentKeyUsageRule>
+  </cpix:ContentKeyUsageRuleList>
+</cpix:CPIX>
+```
+
+### Multikey with audio clear
+
+Get a CPIX 2.1 document that use only 1 content key for video, audio tracks are left on the clear. supported for DASH with PlayReady and Widevine.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
Get a CPIX 2.1 document that uses only 1 content key for video, audio tracks are left on the clear. Only supported for DASH with PlayReady and Widevine.
```

File: docs/source/VUDRM-CPIX.md [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+
+#### optional query params:
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
#### Optional query params:
```

File: docs/source/VUDRM-CPIX.md [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
+
+* `drm` : comma separated list of DRM systems to be included included on the response.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
* `drm` : comma separated list of DRM systems to be included included in the response.
```

File: docs/source/VUDRM-CPIX.md [hunk 10]
```diff
@@ -0,0 +1,228 @@
...
+    <cpix:ContentKeyUsageRule kid="22222222-2222-2222-22222-22222222222">
+      <cpix:AudioFilter></cpix:AudioFilter>
+    </cpix:ContentKeyUsageRule>
+  </cpix:ContentKeyUsageRuleList>
+</cpix:CPIX>
+```
+
+### Keys
+
+Get a encryption keys and license URLs for a given `<content-id>` (JSON response).
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
Get encryption keys and license URLs for a given `<content-id>` (JSON response).
```

File: docs/source/VUDRM-CPIX.md [hunk 11]
```diff
@@ -0,0 +1,228 @@
...
+
+#### optional query params:
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
#### Optional query params:
```

File: docs/source/VUDRM-CPIX.md [hunk 12]
```diff
@@ -0,0 +1,228 @@
...
+
+* `drm` : comma separated list of DRM systems to be included included on the response.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
* `drm` : comma separated list of DRM systems to be included included in the response.
```

File: docs/source/index.rst [hunk 13]
```diff
@@ -20,4 +20,5 @@ Contents:
    VUDRM-token.md
```
Comment (PerritonBranch) [inline docs/source/index.rst]: Could you put this list into alphabetical order?

File: docs/source/VUDRM-CPIX.md [hunk 14]
```diff
@@ -0,0 +1,228 @@
+# CPIX API
+
+You may use our CPIX API in order to fetch VUDRM encyrption keys on CPIX XML document format. Compatible with Unified Streaming (version 1.9.3 and above).
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
You may use our CPIX API in order to fetch VUDRM encyrption keys in CPIX XML document format. Compatible with Unified Streaming (version 1.9.3 and above).
```

File: docs/source/index.rst (hunk 13 above)
Comment (rogerpales) [reply docs/source/index.rst]: Dont you think it should ordered by relevance?, geo-location is far less important than Token.
You never see Contact us and FAQ sections at the top of a menu

Comment (PerritonBranch) [reply docs/source/index.rst]: Thats a not a bad point, @fschurer is probably the person to decide?

Comment (rogerpales) [reply docs/source/index.rst]: Sure yea just did when then i thought about it, I dont really mind that much tbh. Either way, reordering the menu is not what this PR is about, is it?

Comment (PerritonBranch) [reply docs/source/index.rst]: I guess not, but where this documentation come in the list is directly affected by the out come of that discussion.

**Comment by somerobmitchell** [comment]
@rogerpales added do not merge until @fschurer has reviewed and we have agreed to launch our "cpix" integration.

File: docs/source/VUDRM-CPIX.md [hunk 15]
```diff
@@ -0,0 +1,228 @@
+# CPIX API
+
+You may use our CPIX API in order to fetch VUDRM encryption keys in CPIX XML document format. Compatible with Unified Streaming (version 1.9.3 and above).
+
+> Replace `<api-key>`, `<client-name>` and `<content-id>` with appropiate values.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
> Replace `<api-key>`, `<client-name>`, and `<content-id>` with appropiate values.
```

File: docs/source/VUDRM-CPIX.md [hunk 16]
```diff
@@ -0,0 +1,228 @@
...
+    <cpix:ContentKeyUsageRule kid="22222222-2222-2222-22222-22222222222">
+      <cpix:AudioFilter></cpix:AudioFilter>
+    </cpix:ContentKeyUsageRule>
+  </cpix:ContentKeyUsageRuleList>
+</cpix:CPIX>
+```
+
+### Multikey with audio clear
+
+Get a CPIX 2.1 document that use only 1 content key for video, audio tracks are left on the clear. Only supported for DASH with PlayReady and Widevine.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
Get a CPIX 2.1 document that use only 1 content key for video; audio tracks are left on the clear. Only supported by DASH with PlayReady or Widevine.
```

File: docs/source/VUDRM-CPIX.md [hunk 17]
```diff
@@ -0,0 +1,228 @@
...
+        </pskc:Secret>
+      </cpix:Data>
+    </cpix:ContentKey>
+  </cpix:ContentKeyList>
+</cpix:CPIX>
+```
+
+### Multikey
+
+Get a CPIX 2.1 document that uses separate keys for video and audio tracks. Only supported for DASH with PlayReady and Widevine.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-CPIX.md]:
```suggestion
Get a CPIX 2.1 document that uses separate keys for video and audio tracks. Only supported by DASH with PlayReady or Widevine.
```

**Pull request merged**